server/lib/queries.js

```diff
@@ -397,39 +399,41 @@ const getTopSponsors = () => {
 };
 
 /**
- * Returns sponsors ordered by total amount donated
- * (excluding open source collective id 9805)
+ * Returns collectives ordered by burn rate (total amount spent last month)
+ * excluding:
+ * - id 9805 (open source collective host)
+ * - id 1 (opencollective-company)
+ * - id 9804 (wwcode host)
  */
-const getSponsors = memoize(async (where = {}, { orderDirection = 'ASC', limit = 0, offset = 0 }) => {
+const getCollectivesOrderedByTotalAmountSpentLastMonthQuery = async (query) => {
```

Comment:
Can we continue to use the default arguments?

```es6
const getCollectivesOrderedByTotalAmountSpentLastMonthQuery = async ({
  where = {},
  orderDirection = 'ASC',
  limit = 0,
  offset = 0,
}) => {
  // rest of code
};
```

This makes the `query` argument more transparent and cleans up the body of the function.

Reply:
Ah, I thought that

```
const func = ({ limit: 10, offset: 0 }) => {
console.log(limit);
}
```

Would return `undefined` when we call `func({offset: 1})`, but apparently, it's not. So cool, will change.